src-frontend/app/controllers/procedure.js

```diff
@@ -29,16 +29,20 @@ export default Ember.Controller.extend({
             page.save();
         },
 
-        updateSortOrder: function(indices) {
-            this.beginPropertyChanges();
+        updateSortOrder: function(pageModels) {
+            var ctx = this;
```

Comment:
Can we call this `controller` instead of `ctx`? I think it makes it easier to understand